lib/src/domain/model/article.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:json_annotation/json_annotation.dart';
+
+part 'article.g.dart';
+
+@JsonSerializable()
+class Article {
+  final int id;
+
+  final String title;
```

Comment:
trzymamy w domenie publiczne pola?

Reply:
raz, że to nie powinna być domena, a co do pól to powinny być prywatne

Reply:
czemu nie domena?

Reply:
ta konkretna klasa to będzie tylko do serializacji, ogólnie to co wrzuciłem to całkiem pominąłem warstwę domeny, inny task jest na to

Reply:
a, ok :)